draft-ietf-rats-msg-wrap.md

```diff
@@ -587,6 +591,15 @@ As an example, a `cbor-record` containing an UCCS (Unprotected CWT Claims Sets)
 
 This document introduces a format for holding multiple CMW items in a collection.
 If the collection is not protected from tampering by external security measures (such as object security primitives) or internal mechanisms (such as intra-item binding), an attacker could easily manipulate the collection's contents.
+It is the responsibility of the CMW collection producer to ensure that the contents of the collection are integrity-protected.
+In particular, when a CMW is used to carry multiple Evidence messages for a composite device or layered attestation, there should be strong binding between the Evidence messages within the collection.
```

Comment:
```suggestion
In particular, when a CMW is used to carry multiple Evidence messages for a composite device or layered attestation, there should be strong binding between the Evidence messages within the collection. The designer of the attestation technology is typically in charge of ensuring that the security properties are met - not the user of the conceptual message wrapper.
```

Reply:
This one seems to break the natural flow between L595 ("In particular, ...") and L596 ("This is to prevent ...").

Reply:
Fixed. Moved the sentence